terms.adoc

```diff
@@ -0,0 +1,130 @@
+
+Developer ::
+A Committer or Contributor.
+
+Ecosystem ::
+A commercial Ecosystem is a system in which companies, organizations, and individuals all work together for mutual benefit.
+
+Eclipse Management Organization ::
+The Eclipse Management Organization (EMO) consists of the Eclipse Foundation staff, and the Eclipse Architecture and Planning Councils. 
+
+Graduation Review ::
+A Graduation Review is a type of Review that marks a Project's transition from the Incubation Phase to the Mature Phase.
+
+Incubation Branding ::
+Incubation Branding is the branding that must be applied to all projects in the Incubation Phase.
+
+Incubation Phase ::
+The Incubation Phase is a Phase during which the Project Team develops the process, the community, and the technology for a new Project.
+
+Integration Build ::
+An Integration Build is a build of the project content that is generated periodically (generally weekly) and made available to the Developer community for testing purposes.
+
+Major Release ::
+A Major Release is a type of Release that includes either significant new functionality and/or breaking changes.
+
+Mature Phase ::
+The Mature Phase is a Phase during which a Project operates with an open and transparent process, has an actively involved and growing community, and produces Eclipse-quality technology.
+
+Membership at Large ::
+The Membership at Large (or _Membership_) refers to all members of all types, as defined by the {bylawsUrl}[Bylaws of the Eclipse Foundation].
+
+Milestone ::
+A Milestone is a build of the project content intended for limited distribution to demonstrate progress and solicit feedback. 
+
+Minor Release ::
+A Minor Release is a type of Release that includes new features over a Major Release.
+
+Nightly Build ::
+A Nightly Build is a build of the project content that is generated nightly and made available to the Developer community for testing purposes.
+
+Permanent Incubator ::
+A Permanent Incubator is a Project that is intended to perpetually remain in the Incubation Phase.
+
+Phase ::
+A Phase is a stage in the Project lifecycle. See Pre-proposal Phase, Proposal Phase, Incubation Phase, and Mature Phase.
+
+Planning Council ::
+The Eclipse Planning Council is a council that is responsible for cross-project planning, user interface conflicts, and all other coordination and integration issues.
+
+Pre-proposal Phase ::
+The Pre-proposal Phase is a Phase during which a new Project Proposal is created. 
+
+Progress Review ::
+A Progress Review a type of Review that is used by a Project Team to summarize the accomplishments of the Project, verify that the Eclipse Development Process and IP Policy have been followed, and to highlight any remaining quality and/or architectural issues.
+
+Project ::
+A Project (or more specifically, an _Open Source Project_) is the primary functional unit for open source development at the Eclipse Foundation.
```

Comment:
I don't find it very "defining". We should may try to define what a project is made of (code and committers at least)

Reply:
Another thing in this definition would be a note that unless specified, a Project is not a Top-Level Project.

Reply:
We describe what a project is made of elsewhere in the document. I'll admit that I really struggle with a formal definition here. In an abstract sense, a project is where "the work gets done". A formal definition should, I think, be primarily about scope.